Kitodo/src/main/java/de/sub/goobi/forms/LoginForm.java

```diff
@@ -309,7 +306,7 @@ public User getMyBenutzer() {
                 if (newUser != null) {
                     myBenutzer = new User(newUser);
                 }
-                return this.myBenutzer;
+        return this.myBenutzer;
```

Comment:
You are introducing a wrong indent level. Please fix this.

Reply:
Strange. How did this happen?